examples/json-rpc-api-server.py

```diff
@@ -73,13 +72,15 @@ def main():
     ndb = NotificationDB.instance()
     ndb.start()
 
-    api_server = JsonRpcApi(args.port)
+    api_server = JsonRpcApi(settings.NODE_PORT)
```

Comment:
I want to keep this for now, since a user might want to start the json-rpc api on a non-standard port, therefore I don't want to hardcode `settings.NODE_PORT` within `JsonRpcApi` for the `getversion` response.

Reply:
RPC server shouldn't use the node port, as the base network API will be using that.

Reply:
So I guess we should probably add an `RPC` port to the settings file

Reply:
Lol, of course ^^